lib/taskbook.js

```diff
@@ -499,6 +499,18 @@ class Taskbook {
     this._save(_data);
     render.successPriority(id, level);
   }
+
+  clean() {
+    const keysToDelete = [];
+
+    Object.entries(this._data).forEach(([id, item]) => {
+      if (item.isComplete) {
+        keysToDelete.push(id);
+      }
+    });
+
+    this.deleteItems(keysToDelete);
```

Comment:
Before calling `this.deleteItems(ids)` we should check if the `ids` array is actually empty and return if so:

```js
if (ids.length === 0) {
  return;
}

this.deleteItems(ids);
```